### DISCUSSION TITLE
Offset für BME280 Werte

### DISCUSSION THREAD

**Discussion by oliboehm** [pixelit-project/PixelIt#55] [Ideas]
Bei mir weichen Temp und Luftdruck immer von den realen Werten ab. Wäre schön, wenn über das Webinterface Offsets für die BME280 Werte eingeben werden könnten.

**Comment by o0shojo0o** [comment]
@foorschtbar hast du auch solche Probleme mit dem BME280 ?

**Reply by foorschtbar** [reply]
Jein. Das ist bekannt wenn man danach sucht. Je nach China-PCB erzeugt der LDO bissl wärme. Hat mich nicht gestört, aber ein valider punkt wenn es jmd. genau haben möchte.

**Comment by o0shojo0o** [comment]
Ist nun in der aktuellen Git Version implementiert.
